LICENSE.md

```diff
@@ -42,4 +42,5 @@ rickyboy320 <rickw320@hotmail.com>
 DoNotSpamPls <7570108+DoNotSpamPls@users.noreply.github.com>
 JRoy <joshroy126@gmail.com>
 Robert Norman <spottedleaf@spottedleaf.dev>, <Spottedleaf@users.noreply.github.com>
+Callum Seabrook <callum.seabrook@prevarinite.com>
```

Comment:
Incredible.

Reply:
Well, thought I might as well as I'm more than happy to license this under MIT